packages/ses/src/error/assert.js

```diff
@@ -74,39 +78,40 @@ const canBeBare = freeze(/^[\w:-]( ?[\w:-])*$/);
 /**
  * @type {AssertionUtilities['bare']}
  */
-const bare = (payload, spaces = undefined) => {
-  if (typeof payload !== 'string' || !regexpTest(canBeBare, payload)) {
-    return quote(payload, spaces);
+const bare = (text, spaces = undefined) => {
+  if (typeof text !== 'string' || !regexpTest(canBeBare, text)) {
+    return quote(text, spaces);
   }
   const result = freeze({
-    toString: freeze(() => payload),
+    toString: freeze(() => text),
   });
-  weakmapSet(declassifiers, result, payload);
+  weakmapSet(declassifiers, result, text);
   return result;
 };
 freeze(bare);
 
 // /////////////////////////////////////////////////////////////////////////////
 
 /**
- * @typedef {object} HiddenDetails
- *
- * Captures the arguments passed to the `details` template string tag.
+ * @typedef {{ template: TemplateStringsArray | string[], args: any[] }} DetailsParts
```

Comment:
I'm ok with the rename if the doc-comment here mentions that a `DetailParts` may hold redacted secrets, and so must not leak. I originally named it `Hidden*` to suggest that such care should be taken.